kerlescan/view_helpers.py

```diff
@@ -79,10 +79,13 @@ def ensure_has_permission(**kwargs):
             for p in perms:
                 if p in kwargs["permissions"]:
                     return  # allow
-            raise HTTPError(
-                HTTPStatus.FORBIDDEN,
-                message="user does not have access to %s" % kwargs["permissions"],
-            )
+            if perms is None:
+                raise HTTPError(
+                    HTTPStatus.FORBIDDEN,
+                    message="user does not have access to %s" % kwargs["permissions"],
+                )
+            else:
+                return
```

Comment:
This doesn't seem right and I'm not sure why it needed to change.  With this change, the user is allowed access as long as any permission is returned, even if it not a required one.

Reply:
You are right... 
First - I should have put an empty array there, not None....
Second - I would be more happy with returning proper list of permissions there https://github.com/RedHatInsights/kerlescan/pull/21/files#diff-a111b01640aa13b9b6ee1814edf88d83e0a194dae980c80b5b8a0d63cdb66410R19 in the same format as https://github.com/RedHatInsights/kerlescan/pull/21/files#diff-a111b01640aa13b9b6ee1814edf88d83e0a194dae980c80b5b8a0d63cdb66410R28